composer.json

```diff
@@ -32,10 +32,12 @@
         "michelf/php-markdown": "~1.4",
         "mikehaertl/php-shellcommand": "^1.4",
         "phpstan/phpstan": "^0.12",
-        "phpunit/phpunit": "^7.5",
         "scrutinizer/ocular": "^1.5",
-        "symfony/finder": "^4.2"
+        "symfony/finder": "^4.2",
+        "symfony/phpunit-bridge": "^5.1"
```

Comment:
There is currently a bug effecting phpunit bridge which means it doesn't work with composer 2 and phpunit 9.3.

Reply:
https://github.com/symfony/symfony/issues/37601

Reply:
Ah, thanks for that!  I've been trying to follow your work over the past day or two on this but didn't come across that one yet :wink:

Reply:
Yeh, I've mostly not had to deal with the phpunit-bridge package. I came across this bug because Composer itself uses it to run its tests, and ran into the same problem as you: not using PHPUnit 9 yet, but getting match syntax error on PHP 8.

Reply:
phpunit-bridge will help us easily test on PHP 7.1 - 8.0, as those will require different versions of PHPUnit, some of which need `: void()` return types whereas others don't.

Once https://github.com/symfony/symfony/pull/37607 is merged and released we should be able to proceed with this MR here.

Reply:
> phpunit-bridge will help us easily test on PHP 7.1 - 8.0, as those will require different versions of PHPUnit, some of which need : void() return types whereas others don't.

is that actually needed? You can replace setup and teardown methods with your own methods with annotations.

Reply:
Also, adding void where the interface didn't want it is permitted? https://3v4l.org/9me7a

Reply:
This is true, but I'm a lazy developer and would rather avoid extra work and merge conflicts if I can :stuck_out_tongue:

Reply:
Fair enough. Hopefully someone from the Symfony core team will review the PR on Monday.

Reply:
Meh, I have nothing else to do today.  I'll go native PHPUnit and try out the bridge another time.